CONTRIBUTING.md

```diff
@@ -64,33 +64,39 @@ You also need to install other code dependencies not mandatory in the runtime en
 
 #### Add a component to an existing provider (AWS or GCP)
 
-We currently support two providers: Amazon Web Services (AWS) and Google Cloud Platform (GCP). If you want to play around with one of this provider, you can follow the following guidelines:
+We currently support two providers: Amazon Web Services (AWS) and Google Cloud Platform (GCP). If you want to play around with one of this provider, you can follow the following guidelines.
 
-##### GCP
+For both providers, you need to add your component to this places or equivalent:
 
-In order to add a component to GCP provider, you must add your component name (let's suppose `Compute Instance`) to this places:
-
-* As a `const` value [here](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/google/resources.go#L17). :warning: your component name must exactly maps the name as in Terraform documentation. Example: `ComputeInstance` will be later used as `google_compute_instance` :warning:.
+* As a `const` value [here](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/google/resources.go#L17) for Google and [here](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/aws/resources.go#L22) for AWS. :warning: your component name must exactly maps the name as in Terraform documentation. Example: `ComputeInstance` will be later used as `google_compute_instance` :warning:.
 
 ```go
 //go:generate enumer -type ResourceType -addprefix google_ -transform snake -linecomment
 ```
 
 This means you will need to generate some code.
 
-* As a `key/value`, in this [array](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/google/resources.go#L25). This is where you map your component with a middleware function.
+* As a `key/value`, in this [array](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/google/resources.go#L25)([here](https://github.com/cycloidio/terracognita/blob/000789d3bd61b81cf10695d414f4f45346ccc25f/aws/resources.go#L96) for AWS). This is where you map your component with a middleware function.
 
 ```go
 ...
 ComputeInstance: computeInstance,
 ...
 ```
 
-Now, you can write your `computeInstance` function. Check-out the other functions, you basically fetch resources from a middleware layer (`g.gcpr` stands for `GCP reader`) and you add them as Terraform resources.
+Now, you can write your `computeInstance` function. Check-out the other functions, you basically fetch resources from a middleware layer and you add them as Terraform resources.
+
+##### AWS Middleware layer
+
+We are using [RAWS](https://github.com/cycloidio/raws) in order to add abstraction between AWS APIs and Terracognita. This is the [list](https://github.com/cycloidio/raws/issues/8) of all the methods implemented by RAWS. If you do not find the one you are looking for: feel free to submit a PR or to add a comment in the issue.
```

Comment:
That might be deprecated soon, could be worth adding a comment to the issue if not yet done

Reply:
Yeh that's a good idea, I'm finishing some things but the future is to move raws to TC direclty

Reply:
Link https://github.com/cycloidio/terracognita/issues/69